themes/default/views/editor/objects/summary_html_ajax.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/* ----------------------------------------------------------------------
+ * views/editor/objects/summary_html_ajax.php :
+ * ----------------------------------------------------------------------
+ * CollectiveAccess
+ * Open-source collections management software
+ * ----------------------------------------------------------------------
+ *
+ * Software by Whirl-i-Gig (http://www.whirl-i-gig.com)
+ * Copyright 2010-2021 Whirl-i-Gig
+ *
+ * For more information visit http://www.CollectiveAccess.org
+ *
+ * This program is free software; you may redistribute it and/or modify it under
+ * the terms of the provided license as published by Whirl-i-Gig
+ *
+ * CollectiveAccess is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTIES whatsoever, including any implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * This source code is free and modifiable under the terms of
+ * GNU General Public License. (http://www.gnu.org/copyleft/gpl.html). See
+ * the "license.txt" file for details, or visit the CollectiveAccess web site at
+ * http://www.CollectiveAccess.org
+ *
+ * ----------------------------------------------------------------------
+ */
+	$t_item 				= $this->getVar('t_subject');
+	$item_id 				= $this->getVar('subject_id');
+
+	$t_display 				= $this->getVar('t_display');
+	$placements 			= $this->getVar("placements");
+	$reps 					= $t_item->getRepresentations(array("thumbnail", "small", "medium"));
+?>
+    <div id="summary" style="clear: both;">
+<?php
+    print caEditorPrintSummaryControls($this, TRUE);
+    print caEditorFieldList($this->request, $t_item, [], []);
+?>
+	<div id="title">
+		<?= $t_item->getLabelForDisplay(); ?>
+	</div><!-- end title -->
+	<table border="0" cellpadding="0" cellspacing="0" width="100%">
+		<tr class='summaryImages'>
+			<td valign="top" align="center" width="744">
+<?php
+	if (is_array($reps)) {
+		foreach($reps as $rep) {
+			if(sizeof($reps) > 1){
+				# --- more than one rep show thumbnails
+				$padding_top = ((120 - $rep["info"]["thumbnail"]["HEIGHT"])/2) + 5;
```

Comment:
I don't really like this duplicate code but I can be convinced to live with it

Reply:
yeah, I'm not thrilled. If we have time maybe we can look at refactoring instead of just shoving in the old change :crossed_fingers: